packages/engine/paima-runtime/src/runtime-loops.ts

```diff
@@ -82,10 +79,14 @@ async function runPresync(
         .map(([network, h]) => [network, h + stepSize - 1])
     );
 
-    if (upper[Network.EVM] > presyncBlockHeight[Network.EVM]) {
-      doLog(`${JSON.stringify(presyncBlockHeight)}-${JSON.stringify(upper)}`);
-    } else {
-      doLog(`${JSON.stringify(presyncBlockHeight)}`);
+    for (const network of Object.keys(networks)) {
+      if (upper[network] > presyncBlockHeight[network]) {
+        doLog(
+          `${network}: ${JSON.stringify(presyncBlockHeight[network])}-${JSON.stringify(upper[network])}`
+        );
+      } else {
+        doLog(`${network}: ${JSON.stringify(presyncBlockHeight[network])}`);
+      }
     }
```

Comment:
What is this log even supposed to mean? If this is really something we want to show users on startup, we should make it clearer what this is supposed to be

Reply:
The log was already there, it says what's the range that is going to be fetched in that presync round. I guess I could add a `fetching blocks in range: ` to the msg or something.

Reply:
Yeah I know this was added in a previous commit, but every time I launch paima engine I always wonder what this log was meant to be, so best to be more descriptive